django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -1966,30 +2019,50 @@ def test_keyword_search_searching_all_fields(self):
         includes_search_term = "brevity is the soul of wit"
         doesnt_include_search_term = "longevity is the soul of wit"
         source = make_fake_source()
+
+        # For all the chants below: incipit is automatically calculated
+        # based on fulltext every time a chant is saved. In order to set
+        # the incipit, we need to create the chant (.save() is called
+        # at the end of make_fake_chant()), and then update the incipit
+        # to ensure the proper value is stored in the database for when
+        # we make the request for the page.
+
         chant_incipit = make_fake_chant(
             source=source,
-            incipit=includes_search_term,  # <==
             manuscript_full_text=doesnt_include_search_term,
             manuscript_full_text_std_spelling=doesnt_include_search_term,
         )
+        Chant.objects.filter(id=chant_incipit.id).update(
+            incipit=includes_search_term  # <==
```

Comment:
What does this comment mean?

Reply:
I have a hard time visually parsing series of items whose relevant difference is `includes_search_term` vs. `doesnt_include_search_term`. This was my effort to flag the relevant items that should be caught by the view.